### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by ivaan1992**
Made a new css 
Added media queries 
Added linters 
Edited HTML file
Edited README file
Fixed linters issues

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,8 @@ This is my first professional project I'm doing for Microverse.
```
Comment (SajjadAhmad14) [inline README.md:10]: Please add instructions to set the project up locally.

File: style.css [hunk 2]
```diff
@@ -1,414 +1,424 @@
-body {
```
Comment (SajjadAhmad14) [inline style.css:1]: Please use the font color for the content that is already defined in the design. Moreover the value for min-width property should be in range of 992px - 1100px.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Ivan Linares Gaona, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
- [x] GitHub flow is followed.
## To Improve 🙏

- [x] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by carlos-ssh — COMMENTED** [review]
Hi @ivaan1992 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**